lib/metricsImporter/cli.js

```diff
@@ -0,0 +1,224 @@
+const client = require('./client');
+
+function validateOne(option, value) {
+  if (!value) {
+    console.error(`--${ option } is required`);
+    return false;
+  }
+  if (Array.isArray(value)) {
+    console.Error(`Specify only one -${ option }`);
+    return false;
+  }
+  return true;
+}
+
+function validateZeroOrOne(option, value) {
+  if (Array.isArray(value)) {
+    console.error(`SSpecify at most one --${ option }`);
+    return false;
+  }
+  return true;
+}
+
+class MetricsImporterCli {
+  constructor(client) {
+    this.client = client;
+  }
+
+  async routeMethod(args) {
+    const routes = {
+      'importer': {
+        create: (args) => this.importerCreate(args),
+      },
+      source: {
+        'check-query': (args) => this.sourceCheckQuery(args),
+      },
+      'logs': (args) => this.logs(args),
+    };
+
+    let route = routes;
+    let next;
+    while (typeof route === 'object') {
+      next = args._[0];
+      args._.shift();
+      if (!next) {
+        route = null;
+        break;
+      }
+      route = route[next];
+    }
+
+    if (!route) {
+      console.error("Unrecognized command. See metrics-importer help for usage");
+      return;
+    }
+    await route(args);
+  }
+
+  help() {
+    console.warn(`
+    usage: morgue metrics-import [<entity>] <command> <options>
+    
+    Command may be one of the following:
+    
+    source check-query: Run a test query against a given source.
+    importer create: create an importer.
+    logs: Get logs by source id or importer id.
+    
+    For full documentation, please see:
+    https://github.com/backtrace-labs/backtrace-morgue
+    `);
+  }
+
+  async logs(args) {
+    const project = args._[0];
+    const sourceId = args['source'];
+    const importerId = args['importer'];
+    let limit = 100;
+
+    if (!project) {
+      console.log("Project is required");
+      return;
+    }
+    if (Array.isArray(sourceId)) {
+      console.error("Specify only one --source");
+      return;
+    }
+    if (Array.isArray(importerId)) {
+      console.error("Specify only one --importer");
+      return;
+    }
+    if (!sourceId && !importerId) {
+      console.error("Specify one of --source or --importer");
+      return;
+    }
+
+    if ('limit' in args) {
+      limit = Number.parseInt(args.limit);
+      if (Number.isNaN(limit) || limit <= 0) {
+        console.error("--limit must be a positive integer");
+        return;
+      }
+    }
+
+
```

Comment:
Do we have a styling tool for morgue? Might be worth running whatever roughly matches morgue style on this new file.